### PR TITLE
fix: check for null description before appending content

### PR DESCRIPTION
**Pull request by jahn-junior**
- [X] Have you followed the guidelines for contributing?
- [X] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [X] Have you successfully run `make lint && make test`?

---

Resolves https://github.com/canonical/pydantic-kitbash/issues/56

### CODE REVIEW

File: tests/integration/example/index.rst [hunk 1]
```diff
@@ -39,6 +39,10 @@ Field directive
 
     :ref:`References <index-test>` work too!
 
+.. kitbash-field:: example.project.MockModel no_desc
+
+    This field doesn't have a description, so this should take its place.
```
Comment (tigarmo) [inline tests/integration/example/index.rst:44]: can/should you update the test to check for this text in the html?

**Review by tigarmo — APPROVED** [review]

File: tests/integration/example/index.rst (hunk 1 above)
Comment (jahn-junior) [reply tests/integration/example/index.rst]: https://github.com/canonical/pydantic-kitbash/pull/64/commits/2124ca496cbfd062100defddc0ebab211ee52bea

The integration test should honestly be updated to include a few other edge cases as well, but I'll do that in a separate PR.
